### ISSUE TITLE
Inquiry about your predicted person boxes for experiments

### ISSUE THREAD

**Issue by HuangOwen** [facebookresearch/video-long-term-feature-banks#4]
Thanks for your great work! 
I have a question of your predicted person boxes for experiments,  I notice that in ava_train_predicted_boxes.csv there is action label from 1 to 80 while some of them are labeled as -1(no action perhaps?). I was wondering what is the criterion of the 'no action' human instance. Do you pick them out by IOU with GT  boxes of training set? 
Thanks and looking forward to your reply!

**Comment by dingli-dean** [comment]
Maybe "-1" stands for the mismatched boxes when comparing the action labels of detected boxes with that of ground truth?

**Comment by chaoyuaw** [comment]
Hi, thanks for the questions!

We assign labels of a ground-truth box to a predicted box if they overlap with IoU at least 0.9. A predicted box might have no labels assigned. In that case the label is "-1".

**Comment by HuangOwen** [comment]
That's reasonable, thanks for your reply!
